data/input_files/error.log

```diff
@@ -0,0 +1,5 @@
+Error with...
```

Comment:
Please remove this file and log the failing tests in an issue as well just commenting out the failing tests